### ISSUE TITLE
Symfony 4.1 update - Error

### ISSUE THREAD

**Issue by vitalyiegorov** [Lakion/ApiTestCase#123]
@lchrusciel Today after updating to the latest Symfony 4.1 I start getting this error in my tests:
```
Error : Call to undefined method Symfony\Bundle\FrameworkBundle\Test\TestContainer::getMockedServices()
 /vendor/lakion/api-test-case/src/ApiTestCase.php:117
```

Problem related to getting MockedContainer Services for unmocking:
```php
public function tearDown()
    {
        if (null !== $this->client && null !== $this->client->getContainer()) {
            foreach ($this->client->getContainer()->getMockedServices() as $id => $service) {
                $this->client->getContainer()->unmock($id);
            }
        }

        \Mockery::close();
        $this->client = null;
        $this->entityManager = null;
        $this->fixtureLoader = null;

        parent::tearDown();
    }
```

As [symfony update states](https://symfony.com/blog/new-in-symfony-4-1-simpler-service-testing) they have simplified testing container =) So this code should be removed/changed and docs updated.

As temporary solution/workaround until fix would be released create your own TestCase wrapper and override `tearDown()` like this:
```php
use Lakion\ApiTestCase\JsonApiTestCase;

/**
 * TestCase wrapper.
 */
class TestCase extends JsonApiTestCase
{
    public function tearDown()
    {
        \Mockery::close();
        $this->client = null;

        static::ensureKernelShutdown();
    }
}
```

**Comment by pamil** [comment]
Same happening for Sylius, I think it's a bug introduced by Symfony.

**Comment by bendavies** [comment]
@pamil are you going to work on this?

**Comment by loic425** [comment]
@pamil @lchrusciel Is it work now ? cause sylius 1.2.0 is released with symfony 4.1 support.

**Comment by lchrusciel** [comment]
3.1.0 is working with Symfony 4.1
